app/services/datetime.js

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment';
+
+const dateFormat = 'DD MMM YYYY';
+
+export function formatDate(date) {
+  return moment.utc(date).format(dateFormat); //  "19 Jan 2024";
```

Comment:
why use of `utc`?
I think we shoudl excpect the date to be passed in client timezone
```suggestion
  return moment(date).format(dateFormat); //  "19 Jan 2024";
```